Find a functionally equivalent code:android.content.RestrictionEntry.setDescription:COMMENT
Method Modifier: public      
Comment:/**
 * Sets the user-visible description of the entry, as a possible sub-text for the title.
 * You can use this to describe the entry in more detail or to display the current state of
 * the restriction.
 * @param description the user-visible description string.
 */

@@ -1,3 +1,3 @@
 {
-    this.description = description;
+    this.mDescription = description;
 }

